examples/wearable/UIComponents/contents/user-inputs-elements/slider/extended_slider.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<meta content="width=device-width, initial-scale=1, user-scalable=no" name="viewport" />
+	<title>
+		Wearable UI
+	</title>
+	<link href="../../../lib/tau/wearable/theme/default/tau.css" rel="stylesheet" />
+	<link href="../../../lib/tau/wearable/theme/default/tau.circle.css" media="all and (-tizen-geometric-shape: circle)" rel="stylesheet" />
+	<link href="../../../css/style.css" rel="stylesheet" />
+	<link href="../../../css/style.circle.css" media="all and (-tizen-geometric-shape: circle)" rel="stylesheet" />
+	<script src="../../../js/circle-helper.js">
+	</script>
+	<script src="../../../lib/tau/wearable/js/tau.js">
+	</script>
+</head>
+
+<body>
+	<div class="ui-page" id="pageSlider">
+		<div class="ui-content ui-content-padding">
+			<input class="ui-slider" data-buttons="true" id="slider" max="10" name="circleSlider" type="range" />
+			<div class="ui-slider-icon" id="slider-icon" style="opacity: 0.5">
```

Comment:
Should the style be entered in html?
How about adding to a class conditionally in less?

Reply:
@korneliakobiela @DohyungLim 
I think that other slider examples not refered in GUI Slider section could be moved to other section (see Style Guide section in GUI guideline) and we can take care of them later
we should have only just "Slider" example as it is in GUI

Reply:
The only things I've done is to move slider files from one place to another and add file 'user-inputs-elements/index.html'. These files have been reviewed some time ago. Is it really necessary to edit them?

Reply:
@lslachciak done